internal/sql/catalog/catalog.go

```diff
@@ -251,6 +251,7 @@ type Argument struct {
 	Name       string
 	Type       *ast.TypeName
 	HasDefault bool
+	Variadic   bool
```

Comment:
Why did you add this struct key when there is a mode for Variadic that you can use to check?